REST_API_DOCS.md

```
@@ -0,0 +1,1302 @@
+      "title" : "게시글의 제목 입니다.",
+      "content" : "게시글 내용입니다.",
+      "author" : {
+	      "id" : "admin",
+	      "nickname" : "abc"
+    },
+  "created_at": "2026-01-04T12:00:00Z"
+  }
+}
+```
+
+**Response (403 FORBIDDEN)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)